spec/unit/coinbase/transfer_spec.rb

```diff
@@ -331,19 +327,15 @@
 
       before do
         transfer.transaction.sign(from_key)
-        allow(client)
-          .to receive(:eth_getTransactionByHash)
-          .with(transfer.transaction_hash)
-          .and_return(onchain_transaction)
-        allow(client)
-          .to receive(:eth_getTransactionReceipt)
-          .with(transfer.transaction_hash)
-          .and_return(transaction_receipt)
+        allow(transfers_api)
+          .to receive(:get_transfer)
+          .with(transfer.wallet_id, transfer.from_address_id, transfer.id)
+          .and_return(complete_broadcast_model)
```

Comment:
Note the following syntax will return the `broadcast_model` the first time its called and then the `completed_broadcast_model` the second time!

```suggestion
          .and_return(broadcast_model, complete_broadcast_model)
```